doc/howto/network_ovn_setup.md

```diff
@@ -1,33 +1,144 @@
 (network-ovn-setup)=
 # How to set up OVN with LXD
 
+See the following sections for how to set up a basic OVN network, either as a standalone network or to host a small LXD cluster.
+
+## Set up a standalone OVN network
+
+Complete the following steps to create a standalone OVN network that is connected to a managed LXD parent bridge network (for example, `lxdbr0`) for outbound connectivity.
+
+1. Install the OVN tools on the local server:
+
+        sudo apt install ovn-host ovn-central
+
+1. Configure the OVN integration bridge:
+
+        sudo ovs-vsctl set open_vswitch . \
+          external_ids:ovn-remote=unix:/var/run/ovn/ovnsb_db.sock \
+          external_ids:ovn-encap-type=geneve \
+          external_ids:ovn-encap-ip=127.0.0.1
+
+1. Create an OVN network:
+
+        lxc network set <parent_network> ipv4.dhcp.ranges=<IP_range> ipv4.ovn.ranges=<IP_range>
+        lxc network create ovntest --type=ovn network=<parent_network>
+
+1. Create an instance that uses the `ovntest` network:
+
+        lxc init ubuntu:22.04 c1
+        lxc config device override c1 eth0 network=ovntest
+        lxc start c1
+
+1. Run `lxc list` to show the instance information:
+
+   ```
+   +------+---------+---------------------+----------------------------------------------+-----------+-----------+
+   | NAME |  STATE  |        IPV4         |                     IPV6                     |   TYPE    | SNAPSHOTS |
+   +------+---------+---------------------+----------------------------------------------+-----------+-----------+
+   | c1   | RUNNING | 192.0.2.2 (eth0)    | 2001:db8:cff3:5089:216:3eff:fef0:549f (eth0) | CONTAINER | 0         |
+   +------+---------+---------------------+----------------------------------------------+-----------+-----------+
+   ```
+
+## Set up a LXD cluster on OVN
+
 ```{youtube} https://www.youtube.com/watch?v=1M__Rm9iZb8
 ```
 
-This will create a standalone OVN network that is connected to the parent network lxdbr0 for outbound connectivity.
+Complete the following steps to set up a LXD cluster that uses an OVN network.
+The instructions are for a cluster of four servers, where three of the servers run the distributed database for OVN and the OVN controller, and the fourth server runs only the OVN controller.
+See the linked YouTube video for the complete tutorial.
 
-Install the OVN tools and configure the OVN integration bridge on the local server:
+1. Complete the following steps on three of the four machines:
```

Comment:
Lets be specific about which of the servers to run this on (i.e we should make clear which servers are to have which roles).